common/flatpak-dir.c

```diff
@@ -5658,21 +6082,27 @@ flatpak_dir_update (FlatpakDir          *self,
                                                       &gpg_verify_summary, error))
         return FALSE;
 
+      if (!repo_get_remote_collection_id (self->repo, remote_name, &collection_id, NULL))
+        collection_id = NULL;
```

Comment:
Good catch. I was being relaxed about handling invalid configuration here, but there should be no harm in being strict.